### PR TITLE
feat(CI):add extended_query e2e test in CI

### PR DESCRIPTION
**Pull request by ZENOTME**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?
add basic.slt in extended_query, this file test:

basic statement:
- SET 
- EXPLAIN
- CREATE
- DROP
- INSERT 
- SELECT
- VALUES
- SHOW
- DESCRIBE

basic type:
- int2 int4 int8
- float4 float8 numeric
- data time timestamp

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

## Refer to a related PR or issue link (optional)
#4483

### CODE REVIEW

**Comment by BowenXiao1999** [comment]
As we have discussed in https://github.com/singularity-data/risingwave/issues/4483, we do not want too much duplicate test file.

Do you have any thoughts on this?

**Comment by ZENOTME** [comment]
> As we have discussed in #4483, we do not want too much duplicate test file.
> 
> Do you have any thoughts on this?

Yes, we comment ever

> As we have discussed in #4483, we do not want too much duplicate test file.
> 
> Do you have any thoughts on this?

Yes. the ddl is all about functionality(test ‘show’,’create’) and I think it’s not so costly. So I directly reuse it.

**Comment by ZENOTME** [comment]
Or I can extract some content from it to decrease duplicate test.

File: e2e_test/extended_query/basic.slt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+----
+42 42 42
+
+query VII
+select '20:55:12'::time,'2022-07-12'::date,'2022-07-12 20:55:12'::timestamp;
+----
+20:55:12 2022-07-12 2022-07-12 20:55:12
+
+
+
```
Comment (BowenXiao1999) [inline e2e_test/extended_query/basic.slt:87]:
```suggestion
```

delete empty line

**Review by BowenXiao1999 — APPROVED** [review]
LGTM!

**Comment by BowenXiao1999** [comment]
> According the pgwire execute path, I rewrite the basic.slt. The basic.slt is to cover the path of pgwire. There are two kinds of statement, they run different path of pgwire:
> 
> 1. un-query statement: SET,CREATE,INSERT,FLUSH,EXPLAIN,DROP..
> 2. query statement: SELECT,WITH,VALUES,SHOW,DESCRIBE..
>    We also test different type in extended query mode:
>    smallint,int,bigint
>    real,double precision,numeric
>    time,date,timestamp

FYI: Another approach we discussed is to `SET  extended_mode to true` etc in slt, so that we can resue some slt. 

Let's merge this first.

**Pull request merged**